cmd/mzh19b/main.go

```diff
@@ -3,13 +3,21 @@ package main
 import (
 	"flag"
 	"fmt"
-	"github.com/macaron/go-mh-z19b"
+
+	mhz19b "github.com/macaron/go-mh-z19b"
 )
 
 func main() {
+	calibrate := flag.Bool("r", false, "before calibrate MH-Z19B")
 	device := flag.String("device", "/dev/serial0", "specific MH-Z19B")
 	flag.Parse()
 
-	ppm, _ := mhz19b.Read(*device)
+	mhz19b := mhz19b.New(*device)
```

Comment:
GoLand suggested "Variable 'mhz19b' collides with imported package name."
Uh...

golang user like very short variables.
```suggestion
	m := mhz19b.New(*device)
```

or

```suggestion
	co2 := mhz19b.New(*device)
```